index.html

```diff
@@ -165,8 +165,8 @@ <h3 id="heading">Test Run for Firefox RHEL6.5 (Desktop)</h3>
             </div>
         </footer>
         <script>
-            //IMAGE ROTATIONS
-            var imgs = [
+           //Imgs array and other variables needed for the slider logic :)
+			var imgs = [
```

Comment:
Can you check your editor and configure it to use 4 spaces instead of TAB for indentation. Then please make the code below follow the indentation style as the rest of the file. It makes it easier to review.